protoutil/txutils.go

```diff
@@ -203,8 +220,10 @@ func createTx(
 		}
 
 		if !bytes.Equal(a1, r.Payload) {
-			return nil, errors.Errorf("ProposalResponsePayloads do not match (base64): '%s' vs '%s'",
-				b64.StdEncoding.EncodeToString(r.Payload), b64.StdEncoding.EncodeToString(a1))
+			if logger != nil {
+				logger.Infow("ProposalResponsePayloads do not match (base64)", "payload1", b64.StdEncoding.EncodeToString(a1), "payload2", b64.StdEncoding.EncodeToString(r.Payload))
```

Comment:
why Info and not Warn?

Reply:
I was persuaded by this article: https://dave.cheney.net/2015/11/05/lets-talk-about-logging

Reply:
I'm not persuaded by this article :-)
The precedent in Fabric is basically:
- Normal happy path things that admin might like to know about are logged at Info.
- Unexpected errors that admin should know about are logged at Error.
- Expected transaction issues (like this one) that an admin can usually ignore but might still want to know about are logged at Warning.